terragrunt/mainnet/hetzner/terragrunt.hcl

```diff
@@ -0,0 +1,149 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# TERRAGRUNT CONFIGURATION
+# ----------------------------------------------------------------------------------------------------------------------
+# This is a Hetzner specific configuration that provisions components for a desired network (e.g. testnet).
+# Terraform code used by the Terragrunt configuration described here is located at the following path:
+# github.com/cheqd/cheqd-infra/terraform/hetzner/
+
+# ---------------------------------------------------------------------------------------------------------------------
+# Use the open-sourced cheqd Terraform code located in the cheqd-infra repository.
+# ---------------------------------------------------------------------------------------------------------------------
+terraform {
+  source = "git::https://github.com/cheqd/cheqd-infra.git//terraform/hetzner?ref=${include.root.inputs.tag}"
```

Comment:
@nikola-cheqd we can update this path to use `github.com/cheqd/infra` instead of `cheqd-infra`?